Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -176,9 +177,40 @@ void SVGFormattingContext::run(AvailableSpace const& available_space)
     // NOTE: SVG doesn't have a "formatting context" in the spec, but this is the most
     //       obvious way to drive SVG layout in our engine at the moment.
 
+    auto const& active_document = this->context_box()               // the [context] box
+                                      .root()                       // ...of the [root] viewport
+                                      .document()                   // ...of the document
+                                      .page()                       // ...on the page
+                                      .top_level_browsing_context() // ...in the browser
+                                      .active_document();           // the active document (in the browser)
+
+    bool is_standalone_svg_document = (active_document != nullptr)
+        && !active_document->is_initial_about_blank();
```

Comment:
Instead of this mysterious heuristic, how about we just add a flag to `Document`, something like `is_standalone_svg_document()`? :)

Reply:
Sound.
Need to dig into the document creation code (DOM)... :eyes:

Reply:
Sound. Done.
Haha! Non only is it a good idea, `is_decoded_svg()` was already there! :smiley:
Good learning... I hope ...Applause!!...